Content.Client/Access/UI/AgentIDCardBoundUserInterface.cs

```diff
@@ -27,6 +29,11 @@ protected override void Open()
             _window.OnClose += Close;
             _window.OnNameChanged += OnNameChanged;
             _window.OnJobChanged += OnJobChanged;
+            for (var i = 0; i < _window.JobIconButtons.Count; i++)
+            {
+                var key = i;
+                _window.JobIconButtons[i].OnPressed += _ => OnJobIconChanged(_window.StateByIndex[key]);
```

Comment:
I hate that I can't set a value on buttons in a button group  for use in the event.

Reply:
I can have a look when I get home & have an IDE, but I vaguely remember there being some way to do this.